src/server/game/Spells/SpellInfo.cpp

```diff
@@ -4566,6 +4566,15 @@ bool _isPositiveEffectImpl(SpellInfo const* spellInfo, SpellEffectInfo const& ef
             if ((spellInfo->SpellFamilyFlags[0] & 0x20200000) != 0)
                 return false;
             break;
+        case SPELLFAMILY_WARLOCK:
+            switch (spellInfo->Id)
+            {
+                case 111400: // Burning Rush
+                    return true;
```

Comment:
add `SPELL_ATTR4_AURA_IS_BUFF` in `SpellMgr.cpp` instead

Reply:
done